### PR TITLE
Change name of Kafka createTopics to match parameter type

### PR DESCRIPTION
**Pull request by charlesjmorgan**
I propose that we change the type of `io.prestosql.plugin.kafka.TestingKafka#createTopics` to reflect the name of the method

### CODE REVIEW

File: presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/TestingKafka.java [hunk 1]
```diff
@@ -48,12 +48,14 @@ public void close()
         container.close();
     }
 
-    public void createTopics(String topic)
+    public void createTopics(String... topics)
```
Comment (findepi) [inline presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/TestingKafka.java:51]: if we do not need functionality of creating multiple topics at once, it'd be better to 
rename the method to `createTopic(String)`

Comment (losipiuk) [reply presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/TestingKafka.java:51]: +1

Comment (charlesjmorgan) [reply presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/TestingKafka.java]: changed

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**